rclpy/rclpy/time.py

```diff
@@ -40,6 +40,10 @@ def __init__(self, *, seconds=0, nanoseconds=0, clock_type=ClockType.SYSTEM_TIME
     def nanoseconds(self):
         return _rclpy.rclpy_time_point_get_nanoseconds(self._time_handle)
 
+    def seconds_nanoseconds(self):
+        nanoseconds = self.nanoseconds()
+        return (int(nanoseconds / 1e9), nanoseconds % 1e9)
```

Comment:
The 'nanosec' field must of type 'int',  when assign to msg.header.stamp.nanosec.

Reply:
I am pretty sure the result of the modulo operator is already an integer. Or do you have a case where the result ended up being not an integer?